### PR TITLE
Feature/cycling

### PR DESCRIPTION
**Pull request by Dylan-Graham**
- Added cycling weights
- Corrected typo

### CODE REVIEW

File: app/sportEvaluator.py [hunk 1]
```diff
@@ -61,9 +61,11 @@
 
 # region Weights
 
+1 -0.1 -0.1 -0.5 -0.8
```
Comment (zing-rsa) [inline app/sportEvaluator.py]: Me I'm not understanding this line

File: app/api/rating.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ def get_ratings():
     #     wavePeriod=12,
 
     #     cloudCover=0,
-    #     temparature=15,
```
Comment (zing-rsa) [inline app/api/rating.py:29]: what a nerd

File: app/sportEvaluator.py [hunk 3]
```diff
@@ -136,7 +138,19 @@
         "waveSize": .1,
         "cloudCover": -.3,
         "rain": -.5
-    }
+    },
+    "CyclingWeights": {
```
Comment (zing-rsa) [inline app/sportEvaluator.py:140]: Nice, looks sweet

File: app/sportEvaluator.py (hunk 1 above)
Comment (Dylan-Graham) [reply app/sportEvaluator.py]: It's used for quantum computing the weights

**Pull request merged**